### DISCUSSION TITLE
Action to execute Test Studio Tests against Canvas Apps

### DISCUSSION THREAD

**Discussion by garrytrinder** [microsoft/powerplatform-actions#246] [Ideas] [answered]
I'd like to see an action that enables us to automate the execution of tests written in Test Studio against Canvas Apps to provide some quality gates to ensure updates don't break existing functionality.

**Comment by chris-bal** [comment, marked as answer]
Thanks for your feedback, @garrytrinder.  We're actively building out a roadmap for some significant improvements to quality tooling, including enhancements to Test Studio capabilities.  Integrating with actions as you suggested is an item we're looking at as part of this planning.  

Please do let us know any other feedback you have in this area.

**Comment by davidjenni** [comment]
while our mutual teams work on our roadmap, I'm converting this issue to a discussion.
